binding/Binding/Skottie/Animation.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.ComponentModel;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace SkiaSharp.Skottie
+{
+	public unsafe class Animation : SKObject
+	{
+		public enum RenderFlags
+		{
+			SkipTopLevelIsolation = 0x01,
+			DisableTopLevelClipping = 0x02,
+		}
+
+		internal Animation (IntPtr handle, bool owns)
+			: base (handle, owns)
+		{
+		}
+
+		// SK_C_API skottie_animation_t* skottie_animation_make_from_string(const char* data, size_t length);
+		public static Animation Make (string data)
+			=> GetObject (SkiaApi.skottie_animation_make_from_string (data, data.Length));
+
+		// SK_C_API skottie_animation_t* skottie_animation_make_from_stream(sk_stream_filestream_t* stream);
+		public static Animation Make (SKStream stream)
+			=> GetObject (SkiaApi.skottie_animation_make_from_stream (stream.Handle));
+
+		// SK_C_API skottie_animation_t* skottie_animation_make_from_file(const char* path);
+		public static Animation MakeFromFile(string path)
+			=> GetObject (SkiaApi.skottie_animation_make_from_file (path));
+
+		//SK_C_API void skottie_animation_render(skottie_animation_t *instance, sk_canvas_t *canvas, sk_rect_t *dst);
+		public unsafe void Render(SKCanvas canvas, SKRect dst)
+			=> SkiaApi.skottie_animation_render (Handle, canvas.Handle, &dst);
+
+		//SK_C_API void skottie_animation_render_with_flags(skottie_animation_t *instance, sk_canvas_t *canvas, sk_rect_t *dst, skottie_animation_renderflags_t flags);
+		public void Render (SKCanvas canvas, SKRect dst, RenderFlags flags)
+			=> SkiaApi.skottie_animation_render_with_flags (Handle, canvas.Handle, &dst, flags);
+
+		//SK_C_API void skottie_animation_seek(skottie_animation_t *instance, SkScalar t, sksg_invalidation_controller_t *ic);
+		public void Seek (float t, InvalidationController ic)			=> SkiaApi.skottie_animation_seek (Handle, t, ic?.Handle ?? IntPtr.Zero);
```

Comment:
I see the duration is double and the time is float. Can/should these params also be doubles so that someone can do animation.Duration * 0.3 and not have to cast?

Reply:
Makes sense, it'll be easier to use for sure.